multiplayer/examples/tic-tac-toe/index.html

```diff
@@ -1,10 +1,15 @@
 <html>
   <head>
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, minimum-scale=1, maximum-scale=1"
+    />
     <link rel="stylesheet" type="text/css" href="styles.css" />
+    <link href="https://fonts.cdnfonts.com/css/visitor" rel="stylesheet" />
```

Comment:
AFAIK we should not depend on external resources.

Reply:
Yeah, the games are supposed to be self-contained